solidity/random-beacon/test/utils/dkg.ts

```diff
@@ -112,19 +116,27 @@ export async function signAndSubmitArbitraryDkgResult(
     submitterIndex = firstEligibleIndex(ethers.utils.keccak256(groupPublicKey))
   }
 
+  let membersHash: string
+  if (!groupMembersHash) {
+    membersHash = hashDKGMembers(members, misbehavedIndices)
+  } else {
+    membersHash = groupMembersHash
+  }
```

Comment:
Can it be simplified?

```suggestion
  if (!groupMembersHash) {
    groupMembersHash = hashDKGMembers(members, misbehavedIndices)
  }
```

Reply:
it can't be simplified just like in the suggestion above, because eslint blocks reassignment of the param, but we can do a hybrid approach like here 0040b76